snippets/snippets.code-snippets

```diff
@@ -81,6 +81,13 @@
     "body": ["fmt.Errorf(\"${1:%+v}\", ${0:err})"]
   },
 
+  "ctx context.Context": {
+		"prefix": "ctx",
```

Comment:
Can you indent with 2 spaces instead of tabs?

Reply:
:+1: updated